test/WebSites/ApplicationModelWebSite/Startup.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Routing;
```

Comment:
do we have an aspnet/Routing bug about moving `MapRoute` to a namespace users will already be using?